packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx

```diff
@@ -0,0 +1,119 @@
+import { useCallback, useRef, useState } from 'react'
+
+import {
+  accountSelectors,
+  chatActions,
+  decodeHashId,
+  ReactionTypes
+} from '@audius/common'
+import type { ChatMessage } from '@audius/sdk'
+import { IconSave, PopupPosition } from '@audius/stems'
+import cn from 'classnames'
+import dayjs from 'dayjs'
+import { useDispatch } from 'react-redux'
+
+import { useSelector } from 'common/hooks/useSelector'
+import { reactionMap } from 'components/notification/Notification/components/Reaction'
+
+import styles from './ChatMessageListItem.module.css'
+import { ReactionPopupMenu } from './ReactionPopupMenu'
+
+type ChatMessageListItemProps = {
+  chatId: string
+  message: ChatMessage
+  isTail: boolean
```

Comment:
nit: is this for rendering the tail, or whether this is the last message in the list?

Reply:
Rendering tail